app/controllers/admin/promotional_feature_items_controller.rb

```diff
@@ -44,9 +44,16 @@ def destroy
     Whitehall::PublishingApi.republish_async(@organisation)
     redirect_to_feature "Feature item deleted."
   end
-
+  def confirm_destroy; end
 private
-
+  def get_layout
+    design_system_actions = %w[confirm_destroy]
+    if preview_design_system?(next_release: false) && design_system_actions.include?(action_name)
```

Comment:
we don't have a bootstrap implementation for the confirm delete page so i think we can just have 

```
 if design_system_actions.include?(action_name)
  "design_system"
 else
   "admin"
 end
  end
```